src/ontology/mondo-edit.obo

```diff
@@ -464918,6 +464920,2857 @@ is_a: MONDO:0005550 {source="https://orcid.org/0000-0002-5002-8648"} ! infectiou
+intersection_of: MONDO:0005583 {source="https://orcid.org/0000-0002-5002-8648"} ! non-human animal disease
+intersection_of: MONDO:0700097 MONDO:0002049 {source="https://orcid.org/0000-0002-5002-8648"} ! cross-species analog thrombocytopenia
+
+[Term]
+id: MONDO:1010081
+name: autoimmune thrombocytopenic purpura, non-human animal
+def: "Autoimmune thrombocytopenic purpura that occurs in non-human animals." [MONDO:patterns/nonhuman_disease]
+xref: OMIA:001104 {source="MONDO:equivalentTo"}
+is_a: MONDO:0005583 {source="OMIA:001104"} ! non-human animal disease
+intersection_of: MONDO:0005583 {source="https://orcid.org/0000-0002-5002-8648"} ! non-human animal disease
+intersection_of: MONDO:0700097 MONDO:0008558 {source="https://orcid.org/0000-0002-5002-8648"} ! cross-species analog autoimmune thrombocytopenic purpura
+
+[Term]
+id: MONDO:1010082
+name: Evans syndrome, non-human animal
+def: "Evans syndrome that occurs in non-human animals." [MONDO:patterns/nonhuman_disease]
+xref: OMIA:001286 {source="MONDO:equivalentTo"}
+is_a: MONDO:0005583 {source="OMIA:001286"} ! non-human animal disease
+intersection_of: MONDO:0005583 {source="https://orcid.org/0000-0002-5002-8648"} ! non-human animal disease
+intersection_of: MONDO:0700097 MONDO:0016030 {source="https://orcid.org/0000-0002-5002-8648"} ! cross-species analog Evans syndrome
+
+[Term]
+id: MONDO:1010083
+name: Scott Syndrome, non-human animal
+def: "Scott Syndrome that occurs in non-human animals." [MONDO:patterns/nonhuman_disease]
+xref: OMIA:001353 {source="MONDO:equivalentTo"}
+is_a: MONDO:0005583 {source="OMIA:001353"} ! non-human animal disease
+intersection_of: MONDO:0005583 {source="OMIA:001353"} ! non-human animal disease
+intersection_of: MONDO:0700097 MONDO:0009885 {source="OMIA:001353"} ! cross-species analog Scott syndrome
+
+[Term]
+id: MONDO:1010085
+name: factor XIII deficiency, non-human animal
+def: "Factor XIII deficiency that occurs in non-human animals." [MONDO:patterns/nonhuman_disease]
+xref: OMIA:001818 {source="MONDO:equivalentTo"}
+is_a: MONDO:0005583 {source="OMIA:001818"} ! non-human animal disease
+intersection_of: MONDO:0005583 {source="https://orcid.org/0000-0002-5002-8648"} ! non-human animal disease
+intersection_of: MONDO:0700097 MONDO:0002241 {source="https://orcid.org/0000-0002-5002-8648"} ! cross-species analog factor XIII deficiency
+
+[Term]
+id: MONDO:1010086
+name: myeloperoxidase deficiency, non-human animal
+def: "Myeloperoxidase deficiency that occurs in non-human animals." [MONDO:patterns/nonhuman_disease]
+xref: OMIA:002028 {source="MONDO:equivalentTo"}
+is_a: MONDO:0005583 {source="OMIA:002028"} ! non-human animal disease
+intersection_of: MONDO:0005583 {source="OMIA:002028"} ! non-human animal disease
+intersection_of: MONDO:0700097 MONDO:0009694 {source="OMIA:002028"} ! cross-species analog myeloperoxidase deficiency
+
+[Term]
+id: MONDO:1010088
+name: Bernard-Soulier syndrome type C, non-human animal
```

Comment:
In human, "disease ~ type x" is based on the gene affected. Should we have only the species-specific term(s) instead of this generic term based on "general species unspecific" gene? 
Note that these species-specific diseases (based on species-specific genes) could be analogous to the human type x disease, but it is difficult to create an axiom referring to the general "species-not- specific" gene.